src/java.base/share/classes/sun/launcher/resources/launcher.properties

```diff
@@ -232,6 +232,21 @@ The following options are macOS specific:\n\
 \    -Xdock:icon=<path to icon file>\n\
 \                      override default icon displayed in dock\n\n
 
+# Translators please note do not translate the options themselves
+java.launcher.opt.concise.header  =   Usage: java [java options...] <application> [application arguments...]\n\n\
+\Where <application> is one of:\n\
+\  <MainClass>                to execute the main method of a compiled class\n\
+\  -jar <jar-file.jar>        to execute the main class in a JAR archive\n\
+\  -m <module>[/<MainClass>]  to execute the main class of a module\n\
+\  <SourceFile.java>          to compile and execute a single-file program\n\n\
```

Comment:
```suggestion
\  <SourceFile>.java          to compile and execute a single-file program\n\n\
```

Reply:
Or even `<MainClass>.java` to pair with `<MainClass>`

Reply:
That might not be the case in the future, if the file uses an "unnamed class" (https://openjdk.org/projects/amber/design-notes/on-ramp).

Reply:
Also, if you change `<SourceFile.java>` to `<SourceFile>.java` (I guess it is a good idea), then you should also change ` <jar-file.jar>` to `<jar-file>.jar`, or maybe even `<JarFile>.jar` to better align with the style of the other variables.